roles/ceph-mon/tasks/rgw_pool_pgs.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: create rgw pools if rgw_pool is defined
+  shell: |
+    ceph --connect-timeout 5 --cluster {{ cluster }} osd pool {{ item.key }} {{ item.value.pg_num }}
+  changed_when: false
+  failed_when: false
+  with_dict: "{{ rgw_pool }}"
```

Comment:
this task looks useful outside of rgw, maybe we should call it something more generic like `create_pool_pgs.yml`, and use `create_pools` for the dict name?